surrealdb/clients/http.py

```diff
@@ -84,7 +102,9 @@ async def _request(
         uri: str,
         data: Optional[str] = None,
     ) -> SurrealResponse:
-        surreal_response = await self._http.request(method=method, url=uri, data=data)
+        surreal_response = await self._http.request(
+            method=method, url=uri, content=data
```

Comment:
can you add a trailing comma on the end of this line so black multi-lines it?

Reply:
This is what black formatted

Reply:
yes, but adding a trailing comma will cause it to multi-line; pre-commit is set-up to automatically add trailing commas, but i suppose you are not using it